README.md

```diff
@@ -1,6 +1,32 @@
 # Product-Portal
 
-## Installation
+## Project Details
+
+### How do we complete this project?
+
+#### Ver 0.1 :
+  - **General concept:** Create a static website using array of objects (30 items - will be made by xx and yy) (each json item: title,price,link,catgeory…)
+    - navbar will include 6~ categories (which will be connected with the json file to right product category)
```

Comment:
The categories will be names I / we will choose for now it shouldn't be curcial at the moment it just for the purpose of the guideance, will be depends on our json and will be changeable.